### PR TITLE
CUAV Nora/Nora+ bdshot target

### PR DESCRIPTION
**Pull request by andyp1per**
bdshot target for CUAV Nora and Nora+
Modify 42688 setup to support fast sampling rates

### CODE REVIEW

File: Tools/scripts/generate_manifest.py [hunk 1]
```diff
@@ -43,10 +43,9 @@
     'CubeSolo' : ('CubeSolo', '3DR'),
     'CubeGreen-solo' : ('CubeGreen Solo', 'Hex/ProfiCNC'),
     'CUAVv5' : ('CUAVv5', 'CUAV'),
-    'CUAVv5-bdshot' : ('CUAVv5', 'CUAV'),
```
Comment (peterbarker) [inline Tools/scripts/generate_manifest.py:46]: Should we remove old builds of this?

Comment (andyp1per) [reply Tools/scripts/generate_manifest.py:46]: I think so probably

**Review by tridge — APPROVED** [review]

**Comment by andyp1per** [comment]
@rmackay9 @tridge for some reason I missed this for the backport list. Any chance it can go into 4.3.3?

**Comment by rmackay9** [comment]
@andyp1per, sure, added. txs!

**Comment by rmackay9** [comment]
This is included in 4.3.3-rc1

**Pull request merged**